src/rngs/adapter/reseeding.rs

```diff
@@ -147,7 +147,7 @@ where
 {
 }
 
-#[derive(Debug)]
+#[derive(Debug, PartialEq, Eq)]
```

Comment:
I'm not sure this makes sense. We have a reseed-on-fork feature which in a sense violates `Eq` (though in a new process). We also don't guarantee we won't introduce non-deterministic reseeding in the future.

We also have no use for this, since `ThreadRng` shouldn't be comparable (as already mentioned)?